src/main/java/com/example/coursework2/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.example.coursework2.service;
+
+import com.example.coursework2.Question;
+import com.example.coursework2.exeption.QuestionNumberException;
+import com.example.coursework2.interfaces.ExaminerService;
+import com.example.coursework2.interfaces.QuestionService;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+
+    static private final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
зачем static? сервис и так синглтон